ceph/ceph_admin/helper.py

```diff
@@ -2,10 +2,183 @@
 Contains helper functions that can used across the module.
 """
 import logging
+from os.path import dirname
+
+from jinja2 import Template
+
+from ceph.utils import get_node_by_id
 
 LOG = logging.getLogger()
```

Comment:
Would you be able to change it to `LOG = logging.getLogger(__name__)`

Reply:
done.